test/integration/OrderBook.spec.ts

```diff
@@ -210,6 +210,47 @@ describe('nomatching OrderBook', () => {
     expect(() => orderBook['removeOwnOrder'](remainingOrder!.id, order.pairId, 1)).to.throw;
   });
 
+  describe('stampOwnOrder', () => {
+    const ownOrder = () => {
+      return {
+        pairId: 'LTC/BTC',
+        price: 0.008,
+        quantity: 0.00001,
+        isBuy: false,
+        localId: '',
+        hold: 0,
+      };
+    };
+
+    it('has the same id and localId when localId not provided', async () => {
+      const stampedOrder = orderBook['stampOwnOrder'](ownOrder());
+      expect(stampedOrder.id).to.equal(stampedOrder.localId);
+    });
+
+    it('does not have the same id and localId when localId provided', async () => {
```

Comment:
Rather than simply test that they are different, I think it makes more sense to test that the stamped order's local id is equal to the local id we passed in. We could also have a sanity check in place to make sure they are different, but at that point it would be an issue with how we generate the order id.

Reply:
You're right. Updated the spec title and added the assertion.